src/TrackingContext.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Frontkom\CommonBehatDefinitions;
+
+use Behat\Gherkin\Node\TableNode;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+
+/**
+ * Class TrackingContext.
+ *
+ * Provide Behat step-definitions for operations related to tracking like GTM.
+ */
+class TrackingContext extends RawDrupalContext {
```

Comment:
This class should probably not extend the raw drupal context, since that is part of the drupal extension, and technically this can be used without drupal.

I believe what you are looking for is RawMinkContext